src/components/floating-emojis/FloatingEmojis.js

```diff
@@ -30,7 +29,10 @@ const FloatingEmojis = ({
   wiggleFactor,
   ...props
 }) => {
-  const emojisArray = useMemo(() => castArray(emojis), [emojis]);
+  const emojisArray = useMemo(
```

Comment:
I am curious why would emojis ever be not an array. This file desperately needs TS :joy:.